package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apollo-server-lambda",
-  "version": "2.1.0",
+  "version": "2.1.0-error-handling",
```

Comment:
dont need this

Reply:
addressed in f9072e37d55da9a0339e6f7fd955927a9a32a289